grammars/go.cson

```diff
@@ -80,7 +80,7 @@
   }
   {
     'comment': 'Function declarations'
-    'match': '^(\\bfunc\\b)(?:\\s+(\\([^\\)]+\\)\\s+)?([a-zA-Z_]\\w*)(?=\\())?'
+    'match': '^(\\bfunc\\b)(?:\\s+(\\([^\\)]+\\)\\s+)?([\\w&&[^0-9]]\\w*)(?=\\())?'
```

Comment:
I'm afraid intersections [aren't supported by PCRE](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-text&grammar_url=&grammar_text=%27a%27%3A+%27b%27%0D%0AscopeName%3A+%27nah%27%0D%0Apatterns%3A+%5B%7B%0D%0A%09match%3A+%271%3A+%28%3F%21%5B0-9%5D%29%5C%5Cw%27%0D%0A%09name%3A+%27invalid.illegal.matched.nah%27%0D%0A%7D%2C%7B%0D%0A%09match%3A+%272%3A+%5B%5C%5Cw%26%26%5B%5E0-9%5D%5D%27%0D%0A%09name%3A+%27invalid.illegal.matched.nah%27%0D%0A%7D%5D&code_source=from-text&code_url=&code=1%3A+A%0D%0A2%3A+B); only Oniguruma and Java/Python's regex modules.

To remain portable, I suggest using a negative assertion instead:

```diff
-[\\w&&[^0-9]]
+(?![0-9])\\w
```